src/Control/Director.php

```diff
@@ -1053,33 +1053,6 @@ public static function get_environment_type()
         return $kernel->getEnvironment();
     }
 
-
-    /**
-     * Returns the session environment override
-     *
-     * @internal This method is not a part of public API and will be deleted without a deprecation warning
-     *
-     * @param HTTPRequest $request
-     *
-     * @return string|null null if not overridden, otherwise the actual value
-     */
-    public static function get_session_environment_type(?HTTPRequest $request = null)
```

Comment:
This isn't used *anywhere* in core or supported modules. It was added in https://github.com/silverstripe/silverstripe-framework/commit/ca56e8d78e468874b9267c94d8ec75240b6da0ab#diff-983f05f9a107b9bf66a1ed4230e524be82118dd71f4a795529f8794b762c8b47R1091 as a replacement for `CoreKernel::sessionEnvironment()`. It was removed in https://github.com/silverstripe/silverstripe-framework/pull/10456.

It and this method seem to have only existed to support the `isDev` and `isTest` GET variables, which were [intentionally removed in CMS 5](https://docs.silverstripe.org/en/5/changelogs/5.0.0/#api-general).
> `isDev` and `isTest` query string arguments have been removed due to security concerns (see [ss-2018-005](https://www.silverstripe.org/download/security-releases/ss-2018-005/)).